Bottom: 75bb392e4b9400b65a46082c18ca289433b7f465
Top:    af94f1e12ae5e0db33d551dd9347469cdf8d508b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-18 02:22:34 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
